Mathlib/RingTheory/PowerSeries/Evaluation.lean

```diff
@@ -0,0 +1,66 @@
+/-
+Copyright (c) 2024 Antoine Chambert-Loir, María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir, María Inés de Frutos-Fernández
+-/
+
+import Mathlib.RingTheory.MvPowerSeries.Evaluation
+import Mathlib.RingTheory.PowerSeries.PiTopology
+
+/-! # Evaluation of power series
+
+We adapt evaluation of mv power series to the case of power series.
+
+TODO. Unify more precisely; add documentation.
```

Comment:
It's a bit unsatisfying to specialize all the definitions but none of the basic API. I don't want to bother you more since you are in a hurry, but it would be nice to do a subsequent PR adapting the API.

Also, please make sure that all the names match!

Reply:
I wanted to say more and I couldn't find the identification of `Polynomial R` with `MvPolynomial Unit R`!

Reply:
So I filled the API for `PowerSeries`. This forced me to add four lemmas about polynomials and the identification of polynomials in one variable with multivariate polynomials with a `Unique` type of indeterminates.
Two lemmas compare evaluations; two other compare coercions to power series.